src/types/library/library_item.rs

```diff
@@ -112,6 +120,10 @@ pub struct LibraryItemState {
     #[serde(default, rename = "video_id")]
     #[serde_as(deserialize_as = "DefaultOnNull<NoneAsEmptyString>")]
     pub video_id: Option<String>,
+    /// The last stream watched.
+    #[serde(default)]
+    #[serde_as(deserialize_as = "DefaultOnNull<NoneAsEmptyString>")]
+    pub stream: Option<String>,
```

Comment:
Instead of setting the `stream` to `None` on DataStore::Put, we can instead `#[serde(skip_serialization)]` for the field.
- [ ] TODO: Check if this serialization scheme is intended for all use cases of the LibraryItemStatus

- https://github.com/Stremio/stremio-core-web/blob/af66b2eb6bd1aae35e18cdf66e70879100ee7fd5/src/model/serialize_player.rs#L67-L73
- https://github.com/Stremio/stremio-core-web/blob/af66b2eb6bd1aae35e18cdf66e70879100ee7fd5/src/model/serialize_continue_watching_preview.rs#L11-L15